locales/en.default.json

```diff
@@ -190,7 +190,7 @@
       "new_subtotal": "New subtotal",
       "note": "Order special instructions",
       "checkout": "Check out",
-      "empty": "Your cart is empty",
+      "empty": "Your cart is empty.",
```

Comment:
```suggestion
      "empty": "Your cart is empty",
```
@gretchen-willenberg I know this is a complete sentence but it is used as a heading and I believe we don't use period on these, right?

Reply:
+100 Thank you! Yes. No period. I realized what I had said as I was falling asleep last night and fixed the copy doc this morning.